static/assets/conversations/prologue01.json

```diff
@@ -654,26 +681,20 @@
             },
             {
                 "conditions": {
-                    "varAskedNewSpecies": "true",
-                    "varAskedStationZero": "true",
-                    "varAskedWhyNot": "true"
+                    "seen": "34 35 36"
```

Comment:
Will this pose a problem as we surgically insert conversation nodes? I'm assuming Twine doesn't do this.